core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5346,6 +5346,14 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.IGNORE)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey FUSE_TMP_FOLDER =
```

Comment:
prefer `dir` to `folder`
```suggestion
  public static final PropertyKey FUSE_TMP_DIR=
```